korman/operators/op_mesh.py

```diff
@@ -26,6 +26,13 @@ def poll(cls, context):
         return context.scene.render.engine == "PLASMA_GAME" and context.mode == "OBJECT"
 
 
+FLARE_MATERIAL_BASE_NAME = "FLAREGEN"
+
+def _flare_materials(self, context):
+    matlist = [mat for mat in bpy.data.materials if mat.name.startswith(FLARE_MATERIAL_BASE_NAME)]
+    if matlist:
+        return [(mat.name, mat.name, "", "MATERIAL", idx) for idx, mat in enumerate(matlist)]
```

Comment:
If at all possible, avoid including a numerical index in this enum property. In the best case, you're just going to see the odd behavior where either the getter or setter expects an `str` and the other expects an `int`. At worst, the ability of the operator to recall the previous selection is impaired due to Blender saving the index into matlist, which is potentially nondeterminant.